design/ImageBitmap.md

```diff
@@ -0,0 +1,36 @@
+# WebGPU + ImageBitmap
+
+```webidl
+dictionary ImageBitmapCopyView {
+    ImageBitmap imageBitmap;
+    GPUOrigin3D origin;
+};
+
+partial interface GPUCommandEncoder {
+    void copyImageBitmapToTexture(
+        ImageBitmapCopyView source,
+        GPUTextureCopyView destination,
+        GPUExtent3D copySize);
+};
+```
+
+`copyImageBitmapToTexture` encodes a copy from a source sub-rectangle of an `ImageBitmap` into a destination sub-resource of a `GPUTexture`.
+The `ImageBitmap` is referenced at the time the command buffer is submitted.
```

Comment:
JG: If ImageBitmap is read-only, just say "if it's detached at submit, it's an error"